README.md

```diff
@@ -135,6 +135,72 @@ Options:
   - **full**: includes timezone. e.g. `'1:25 PM EST'`
   - **medium** or **short**: excludes timezone e.g. `'1:25 PM'`
 
+## Fuzzy Date/Time Formatting
+
```

Comment:
Now that there's actually a web standard, I think I'd be good with aligning with them and calling this whole concept "relative date/time formatting" instead of "fuzzy". We only picked that term back before the standard existed and I don't think it's really caught on.

Reply:
Ok. I did like `fuzzy` just because it was different than relative. What we are doing above the web standard is to choose the units automatically, and I think it's good to call it out in the naming. Whether we choose fuzzy or something else.